### PR TITLE
add PRivaCY Coin (PRCY)

### PR DESCRIPTION
**Pull request by lyricidal**
GitHub: https://github.com/PRCYCoin
Website: https://www.prcycoin.com/

CoinPaprika: https://coinpaprika.com/coin/prcy-privacy-coin/
CoinGecko: https://www.coingecko.com/en/coins/prcy-coin
Nomics: https://nomics.com/assets/prcy-privacy-coin

Currently only the BEP20, ERC20 (ETH/POLYGON) versions

### CODE REVIEW

**Comment by tonymorony** [comment]
thank you for the pull request opening @lyricidal 

at first assets you add to GUI should be added into coins repo (source of information about assets supported by protocol)
https://github.com/KomodoPlatform/coins#0-the-coin-must-have-participated-in-a-successful-atomic-swap-using-atomicdex-api
GUI uses coins file from this repo so without that your asset will not work.

here is a draft of the documentation on the desktop GUI asset adding process (will be published soon):
https://github.com/KomodoPlatform/developer-docs/blob/smk-dev/docs/basic-docs/atomicdex/atomicdex-desktop/add-coin-to-atomicdex-desktop.md

**Comment by lyricidal** [comment]
> thank you for the pull request opening @lyricidal
> 
> at first assets you add to GUI should be added into coins repo (source of information about assets supported by protocol) https://github.com/KomodoPlatform/coins#0-the-coin-must-have-participated-in-a-successful-atomic-swap-using-atomicdex-api GUI uses coins file from this repo so without that your asset will not work.
> 
> here is a draft of the documentation on the desktop GUI asset adding process (will be published soon): https://github.com/KomodoPlatform/developer-docs/blob/smk-dev/docs/basic-docs/atomicdex/atomicdex-desktop/add-coin-to-atomicdex-desktop.md

Thank you for the quick response - I had just stumbled upon that as well and am in the process of sorting that out too. Will link to it when complete.

**Comment by lyricidal** [comment]
Submitted PR: https://github.com/KomodoPlatform/coins/pull/429

**Review by smk762 — APPROVED** [review]
Launches and enables as expected

File: assets/config/0.5.4-coins.json [hunk 1]
```diff
@@ -8560,6 +8560,58 @@
     "active": false,
     "currently_enabled": false
   },
+  "PRCY-BEP20": {
+    "coin": "PRCY-BEP20",
+    "name": "PRivaCY Coin",
+    "coinpaprika_id": "prcy-privacy-coin",
+    "coingecko_id": "privacy-coin",
+    "nomics_id": "prcy-privacy-coin",
```
Comment (cipig) [inline assets/config/0.5.4-coins.json:8568]: the nomics_id of PRCY, see https://nomics.com/assets/prcy-privacy-coin/widget (`data-base="PRCY"`)

**Review by cipig — APPROVED** [review]

**Review by smk762 — APPROVED** [review]

**Pull request merged**